src/components/Gallery.css

```diff
@@ -0,0 +1,22 @@
+.column {
+  float: left;
+  padding: 10px;
+  width: calc(12.5% - 20px);
+}
```

Comment:
Are these class names automatically namespaced? If not, we should do that

Reply:
so wold something like `.ix-column` work?

Reply:
Yeah that would be better 👍